crates/biome_js_analyze/src/lint/nursery/use_import_extensions.rs

```diff
@@ -64,6 +70,40 @@ declare_rule! {
     /// ```js
     /// require("./foo.js");
     /// ```
+    ///
+    /// ### Options
+    ///
+    /// Use the options to specify the correct import extensions for your project based on the linted
+    /// file extension. These mappings will override the rule's default logic.
+    ///
+    /// Mainly, this is a temporary workaround that allows Biome to propose correct import extensions
+    /// for TypeScript projects that use ES Modules. TypeScript requires you to specify imports to
+    /// the actual files used in runtime: `.js` or `.mjs` (see more here: https://github.com/microsoft/TypeScript/issues/49083#issuecomment-1435399267).
```

Comment:
I would move this paragraph **after** the code block of the options. Also, add a `:::caution` block, so the docs will render this with [a warning block](https://starlight.astro.build/guides/authoring-content/#more-aside-types):

```rs
/// :::caution
/// Mainly, this is a temporary ...
/// :::
```

Reply:
Reordered and added the warning block around.